docs/release-notes/2.30.0.md

```diff
@@ -0,0 +1,132 @@
+
+## Release Highlights
+
+
+#### New Experimental dashboards
+
+!!! caution alert alert-warning "Important"
+    These experimental dashboards are subject to change. It is recommended to use these dashboards for testing purposes only.
+
+- We have introduced a more user-friendly and insightful experimental Home Dashboard in PMM version 2.30.0. With this new Home Dashboard, you can experience a boost in performance while monitoring a large number of nodes or services in PMM. Repeated rows (for a large number of nodes) on the dashboard can take a considerable time to load the Home Dashboard UI. Our new experimental dashboard ensures fast retrieval of data on the Home Dashboard.
+
+- We have introduced the following new real-time experimental dashboards to collect more metrics and data for MongoDB:
+
+    - **Experimental MongoDB Collection Overview**
+        
+        This dashboard contains panels for data about the Hottest Collections in the MongoDB database. For more information, see [documentation](https://docs.percona.com/percona-monitoring-and-management/details/dashboards/dashboard-mongodb-experimental_collection_overview.html).
+
+    - **Experimental MongoDB Collection Details**
+
+        This experimental dashboard provides detailed information about the top collections by document count, size, and document read for MongoDB databases. For more information, see [documentation](https://docs.percona.com/percona-monitoring-and-management/details/dashboards/dashboard-mongodb-experimental_collection_details.html).
+
+    - **Experimental MongoDB Oplog Details**
+
+        This real-time dashboard contains Oplog details such as Recovery Window, Processing Time, Buffer Capacity, and Oplog Operations. [documentation](https://docs.percona.com/percona-monitoring-and-management/details/dashboards/dashboard-mongodb-experimental_oplog.html).
+
+
+### Database as a Service (DBaaS)
+
+Enhanced the user experience by simplifying the configuration and deployment of DBaaS database clusters:
+
+- With PMM 2.30.0, you can create a DB cluster literally at the click of a button. All the fields will be automatically populated with the default value. Click *Register*, and your DB cluster is created.
+
+- Starting with PMM 2.30.0, we have simplified the DBaaS configuration and deployment of database clusters by providing suggestions on the k8s cluster name based on the kubeconfig file.
+
+- We have simplified DBaaS and Percona Platform connection configuration by suggesting and automatically setting the public address of the PMM server if it's not set up. This happens when connecting to Percona Platform, adding a Kubernetes (K8s) cluster, or adding a DB cluster.
+
+
+### Operators support
+
+- PMM now supports Percona Operators MySQL (1.11) and MongoDB (1.11).
+
+- PMM now supports MongoDB version 5.0.
```

Comment:
We don't support MongoDB 5 in dbaas yet https://jira.percona.com/browse/PMM-9556.